plugins/versionpress/src/Git/CommitMessage.php

```diff
@@ -57,6 +57,19 @@ public function getSubject()
     }
 
     /**
+     * Returns commit message subject without prefix set in constant VP_COMMIT_MESSAGE_PREFIX
+     * 
+     * @return string
+     */
+    public function getUnprefixedSubject()
```

Comment:
@JanVoracek: if result of will be removal of those getters, it will make our domain more anemic, but ok.